.ci.yaml

```diff
@@ -281,6 +281,54 @@ targets:
       target_file: web_build_all_packages.yaml
       channel: stable
 
+  - name: Linux_web web_platform_tests_shard_1 master
```

Comment:
Should the name contain `shard_1/2` or similar?

Reply:
We deliberately don't do that in this repo based on some excellent feedback from Jenn. Renaming is (unlike Cirrus) a giant pain in LUCI, because it has to be done as a multi-step process. So with that kind of naming, the process for going from say, 5 to 6 shards is:

- Add 6 completely new shards in bringup mode
- Wait a while after landing
- Enable all 6 new shards and remove all 5 old shards

With this naming, we just add one shard, and then switch the properties on all the existing ones in a follow-up.

Reply:
> * Add 6 completely new shards in bringup mode
> * Wait a while after landing
> * Enable all 6 new shards and remove all 5 old shards

😱